runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -237,5 +268,17 @@ private void renderZalcanoAOE(Graphics2D graphics, int polySize, String text, Co
 		}
 	}
 
+	private void renderPlayerCount(Graphics2D graphics)
+	{
+		Rectangle rect = new Rectangle(client.getCanvasWidth() / 2 - 15, 0, 40, 25);
+		int[] xpoints = {rect.x, rect.x + rect.width, rect.x + rect.width, rect.x};
+		int[] ypoints = {rect.y, rect.y, rect.y + rect.height, rect.y + rect.height};
+
+		graphics.setFont(new Font("Comic Sans", Font.BOLD, 20));
+		OverlayUtil.renderFilledPolygon(graphics, rect, new Color(0, 0, 0));
+		String playerCount = java.lang.Integer.toString(plugin.getPlayers().size());
+		OverlayUtil.renderTextLocation( graphics, new Point(client.getCanvasWidth() / 2, 20), playerCount, new Color(255, 255, 255));
```

Comment:
`Color.WHITE`

Reply:
Is there advantage using it this way? They are declared somewhere anyway?